src/Workspaces/Core/Portable/Classification/ClassificationContext.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Classification
+{
+    /// <summary>
+    /// A context for aggregating classifications from one or more <see cref="ClassificationProvider"/>'s.
+    /// </summary>
+    internal class ClassificationContext
+    {
+        private readonly List<ClassifiedSpan> _spans;
+        private readonly HashSet<ClassifiedSpan> _spanSet;
+
+        internal ClassificationContext(List<ClassifiedSpan> classifiedSpans, HashSet<ClassifiedSpan> spanSet)
+        {
+            _spans = classifiedSpans;
+            _spanSet = spanSet;
+        }
+
+        public void AddClassification(ClassifiedSpan classifiedSpan)
```

Comment:
I think, before we make this public, we shouldl just make this be 'TaggedText'.  Or possibly TaggedSpan.  I think using 'Tags' here makes sense so we have a single place where we define all the tags.  Then, internally, we could copy this over to the editor terminology.

What do you think?